lab_2/main.py

```diff
@@ -88,8 +125,14 @@ def calculate_plagiarism_score(lcs_length: int, suspicious_sentence_tokens: tupl
     :param suspicious_sentence_tokens: a tuple of tokens
     :return: a score from 0 to 1, where 0 means no plagiarism, 1 – the texts are the same
     """
-    pass
-
+    if not isinstance(lcs_length,int) or not isinstance(suspicious_sentence_tokens,tuple)\
+            or lcs_length<0 or not all(isinstance (i,str) for i in suspicious_sentence_tokens) \
+             or  isinstance(lcs_length,bool) or lcs_length>100 or (not suspicious_sentence_tokens and lcs_length==0):
```

Comment:
lcs_length > 100 is not really good, what if you have really big texts?
you can compare lcs length with something else, another length for example...